timescaledb/overview/release-notes/changes-in-timescaledb-2.md

```diff
@@ -72,9 +72,9 @@ to customize behaviors when needed.**
 For example, in TimescaleDB 2.0 we've separated the automation of _refreshing_ continuous aggregate
 data from the core functionality, giving users the option to manually refresh the data and, if
 desired, add automation via a policy. This also makes this feature consistent with TimescaleDB's
-other policy-driven features – such as retention, reordering, and compression – which offer both
+other policy-driven features—such as retention, reordering, and compression—which offer both
 manual control and automation via policies. We have also opened up our jobs scheduling framework
```

Comment:
```suggestion
manual control and automation using policies. We have also opened up our jobs scheduling framework
```

I know via isn't technically a Latin abbreviation, but it's close enough that it makes me twitch. It also is extremely imprecise in definition, you can usually pick a more precise word, like "through", or "using", etc.